src/FSharp.Analyzers.Cli/FSharp.Analyzers.Cli.fsproj

```diff
@@ -1,20 +1,34 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
+  
   <PropertyGroup>
+    <RollForward>LatestMajor</RollForward>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackAsTool>true</PackAsTool>
     <PackageId>fsharp-analyzers</PackageId>
     <ToolCommandName>fsharp-analyzers</ToolCommandName>
-    <RollForward>LatestMajor</RollForward>
+    <Version>0.11.0</Version>
+    <PackageReleaseNotes>https://github.com/ionide/FSharp.Analyzers.SDK/blob/master/RELEASE_NOTES.md</PackageReleaseNotes>
```

Comment:
Putting a URL here has a bad experience on NuGet.org - instead please use something like [this MSBuild target](https://github.com/dotnet/sdk-container-builds/blob/main/packaging/package.csproj#L37-L42) to read the RELEASE_NOTES.md file and put its contents directly in the release notes of the package.  This will let people view the release notes entirely from NuGet.org.

This is one thing that Ionide.KeepAChangelog does really nicely (centralize the versioning/release notes part of releasing) and it's something I want in every package I maintain.

Reply:
I think Ionide.KeepAChangelog looks like a great tool. Do you want me to add that instead of the target you link to? And would Ionide.KeepAChangelog require renaming `RELEASE_NOTES.md` to `CHANGELOG.md`?

Reply:
Moving to Ionide.KeepAChangelog would be great, but it comes with two caveats:

* the existing README.md file would need to be converted to the Conventional Changelog format in its entirety. This is a bit of a pain, so I made a FAKE task to help with it when I [converted FsAutoComplete](https://github.com/fsharp/FsAutoComplete/pull/881/commits/af8118729e9737071c543de58223164134051fe4#diff-5ebd60ceb88910043c04c5a665d836197d7184488edcf52f590e8fcad6101519R196-R208)
* Ionide.KeepAChangelog has a feature gap where it will not load in Visual Studio at all (because VS using .NET framework MSBuild and has assembly load errors I haven't figured out yet).  Moving to it would mean we'd either forbid VS users from contributing to this project, or (more likely) we'd need a `Condition` on the PackageReference to only add it if `'$(BuildingInsideVisualStudio)' != 'true'`

Reply:
It works fine in VS 17.4.2. I will convert the changelog and add Ionide.KeepAChangelog to this PR.

Reply:
Note that I assume you meant converting the changelog (not the readme) to the "Keep a changelog" format (not "Common  changelog", which seems to be [something else](https://github.com/conventional-changelog/conventional-changelog)).